Makefile

```diff
@@ -504,6 +504,11 @@ java-examples:
 	cd $(SWIFTNAV_ROOT)/java/example && gradle clean fatjar
 	$(call announce-end,"Finished building fat jar")
 
+javadoc javadocs:
```

Comment:
We could make the dependency of the `Java` target so that it runs locally-- would there be any downsides to that?

Reply:
If errors are rare then this is a good place to verify, making everyone check it locally sounds unnecessary